media/js/src/chat.js

```diff
@@ -1,4 +1,59 @@
-$(function() {
+/* jscs:disable requireCamelCaseOrUpperCaseIdentifiers */
+
+require.config({
+    map: {
+        // This configures jquery to not export the $ and jQuery global
+        // variables.
+        '*': {
+            'jquery': 'jquery-private'
+        },
+        'jquery-private': {
+            'jquery': 'jquery'
+        }
+    },
+    paths: {
+        // Major libraries
+        jquery: '../libs/jquery/jquery-min',
+        'jquery-private': '../libs/jquery/jquery-private',
+
+        // Require.js plugins
+        text: '../libs/require/text',
+        domReady: '../libs/require/domReady'
+    },
+    urlArgs: 'bust=' +  (new Date()).getTime()
+});
+
+require([
+    'domReady',
+    'jquery',
+    '../libs/commonmark.min',
+    '../libs/linkify/linkify.min',
+    '../libs/linkify/linkify-html.min',
+    '../libs/js-emoji/emoji.min'
+], function(domReady, $, commonmark, linkify, linkifyHtml, emoji) {
+
+    var renderMarkdown = function(text) {
```

Comment:
I'm turning this into its own utility class on another branch so we don't have to duplicate this code.